### ISSUE TITLE
Add Benefit Type to issue text on add issues page

### ISSUE THREAD

**Issue by leikkisa** [department-of-veterans-affairs/caseflow#9980]
### Story
On the Appeal Add Issues page, a user can select a Benefit Type for issues, but the Benefit Type does not show up in the issue text on the add issues page

### Acceptance criteria
Show the Benefit Type for each issue in the Requested Issues section, before the Issue Category

**Comment by rutvigupta-design** [comment]
@leikkisa do you think this is still relevant or needed?

**Comment by leikkisa** [comment]
@rutvigupta-design Yes. I don't know that it needs design (unless we want to create a new design for how to show issues).  I think when I wrote this, I was thinking a copy change like: "Compensation - Apportionment - description".  Right now it shows "Apportionment - description".

**Comment by GabbyTISTA** [comment]
Moved to https://vajira.max.gov/browse/CASEFLOW-1686
